doc/api/modules.md

```diff
@@ -850,11 +850,28 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.makeRequire(filename)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string|URL} Filename for relative importing
+* Returns: {[`require`][]} Require function
+
+<!-- eslint-skip -->
+```js
+// create a require function which will resolve relative
+// to the current ES Module file
+import Module from 'module';
+const require = Module.makeRequire(new URL(import.meta.url));
```

Comment:
Is there a way to support simply passing the url string around.
Smoothing that over to just work without having to instantiate the URL constructor for things like this would be nice.

Reply:
its done like that to mirror how `fs` allows `URL` objects, i would rather not diverge from that